experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -24,55 +26,49 @@ def __init__(self, application: str, channel: str):
     def create_loader(cls, application: str, channel: str):
         return cls(application, channel)
 
-    def file_path(self):
+    def file_path(self, version: NimbusFeatureVersion = None):
         """Get path to release feature manifest from experimenter (local)."""
+
         if self.application is not None:
-            path = Path(self.MANIFEST_PATH, self.application, f"{self.channel}.fml.yaml")
+            path = Path(self.MANIFEST_PATH, self.application)
+            if version:
+                path /= f"v{version}"
+            path /= f"{self.channel}.fml.yaml"
+
             if Path.exists(path):
                 return path
+            else:
+                logger.error(f"Nimbus FML Loader: Invalid manifest path: {path}")
+                return None
         else:
             logger.error(
                 "Nimbus FML Loader: Invalid application. Failed to get local "
                 "feature manifest."
             )
         return None
 
-    # Todo: Add versioning https://mozilla-hub.atlassian.net/browse/EXP-3875
-    def _get_fml_clients(self) -> list[FmlClient]:
```

Comment:
No longer necessary since we will handle only one version at a time in the loader.